### PR TITLE
basic theming and logo placed

### PR DESCRIPTION
**Pull request by manishagupta90**

### CODE REVIEW

File: themes/custom/srijan_devportal_theme/css/style.css [hunk 1]
```diff
@@ -2,3 +2,72 @@
  * If using a CSS pre-processor, tell it to overwrite this file. If not, place
  * your custom CSS modifications here.
  */
+
+ body {
+  font-family: 'Poppins', sans-serif;
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/css/style.css]: Please rectify indentation.

File: themes/custom/srijan_devportal_theme/css/style.css [hunk 2]
```diff
@@ -2,3 +2,72 @@
...
+
+.footer nav {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.footer nav .nav>li>a {
+  padding: 0;
+  color: #fff;
+}
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/css/style.css]: Add a new line at EOF

File: themes/custom/srijan_devportal_theme/css/style.css [hunk 3]
```diff
@@ -2,3 +2,72 @@
...
+
+/* header-account-nav */
+.header-account-nav {
+  float: right;
+}
+
+ /* footer */
+ .footer {
+  border: none;
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/css/style.css]: Rectify Indentation.

File: themes/custom/srijan_devportal_theme/css/style.css [hunk 4]
```diff
@@ -2,3 +2,72 @@
...
+  border-radius: 0;
+  border: none;
+  background-color: transparent;
+  padding: 0;
+  min-height: auto;
+  margin-bottom: 0px;
+}
+
+/* herader navbar */
+.navbar-nav>li>a {
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/css/style.css]: Please follow coding standards.

File: themes/custom/srijan_devportal_theme/css/style.css (hunk 1 above)
Comment (Manishasrijan) [reply themes/custom/srijan_devportal_theme/css/style.css]: done

File: themes/custom/srijan_devportal_theme/css/style.css (hunk 2 above)
Comment (Manishasrijan) [reply themes/custom/srijan_devportal_theme/css/style.css]: done

File: themes/custom/srijan_devportal_theme/css/style.css (hunk 3 above)
Comment (Manishasrijan) [reply themes/custom/srijan_devportal_theme/css/style.css]: done

File: themes/custom/srijan_devportal_theme/css/style.css (hunk 4 above)
Comment (Manishasrijan) [reply themes/custom/srijan_devportal_theme/css/style.css]: done

**Pull request merged**